### DISCUSSION TITLE
Proper continuous integration pipeline thanks to PowerQueryNet!

### DISCUSSION THREAD

**Discussion by sio** [gsimardnet/PowerQueryNet#27] [Show and tell]
I have successfully implemented a continuous integration pipeline for my Power Query project thanks to PowerQueryNet!

Here it is:

- [Entrypoint and some configuration](https://github.com/sio/LibPQ/tree/master/CI)
- Pipelines:
  - [GitHub Actions workflow file](https://github.com/sio/LibPQ/blob/master/.github/workflows/test.yml)
  - [Sample pipeline failure](https://github.com/sio/LibPQ/runs/3141224632?check_suite_focus=true)
  - [Sample pipeline success](https://github.com/sio/LibPQ/runs/3141444141?check_suite_focus=true)

I'm extremely happy with how straightforward the process was and how little modification of my Power Query code was required! Thank you very much!

**Comment by edurod18** [comment]
how did you get the microsoft mashup reference?
